autoscheduler/frontend/src/.eslintrc.js

```diff
@@ -57,5 +57,6 @@ module.exports = {
         ["in", "instanceof"],
       ],
     }],
+    "@typescript-eslint/no-explicit-any": 0,
```

Comment:
There are times when you are okay with a variable having any type. Specifically, I need them to test Thunk actions in `Redux.test.ts`